pkg/nodebootstrap/assets/bootstrap.al2.sh

```diff
@@ -1,7 +1,31 @@
-#!/bin/sh -eu
+#!/bin/bash -e
```

Comment:
👍 

I agree we should be explicit about what we're actually using.

Reply:
Once we switched to bash here, let's get rid of `-e` and use more readable syntax:
```
set -o errexit
set -o pipefail
set -o nounset
```